stock_landed_costs_purchase_auto/tests/test_purchase_order.py

```diff
@@ -17,22 +18,67 @@ def test_order_lc(self):
         self.order.button_confirm()
         self.assertTrue(self.order.landed_cost_ids)
         picking = self.order.picking_ids
+        # Guarda el ID del primer picking antes de validar
+        picking_id = picking.id
+
+        # Valida el primer picking (solo una unidad)
+        for move in picking.move_ids:
+            move.quantity = 1
         self._action_picking_validate(picking)
+
+        # Refresca el pedido y busca el nuevo picking
+        self.order = self.order.with_env(self.env)
+        new_picking = self.order.picking_ids.filtered(lambda p: p.id != picking_id)
+        self._action_picking_validate(new_picking)
```

Comment:
In general, I don't understand the changes in the tests (or in common.py). Compared to 16.0 https://github.com/OCA/stock-logistics-workflow/blob/16.0/stock_landed_costs_purchase_auto/tests/test_purchase_order.py#L16, the processes to be tested should be the same, right? (1 picking https://github.com/OCA/stock-logistics-workflow/blob/16.0/stock_landed_costs_purchase_auto/tests/test_purchase_order.py# L16C9-L16C22 and several pickings https://github.com/OCA/stock-logistics-workflow/blob/16.0/stock_landed_costs_purchase_auto/tests/test_purchase_order.py#L23) Therefore, I do not understand the need for these changes.